### PR TITLE
Removed transmutable recipe code which was moved to fluo-recipes

### PR DESCRIPTION
**Pull request by mikewalch**
and renamed to AccumuloExport and AccumuloExporter.  Modified export
queue code to use new recipes. Modified UriCountExport and
DomainExport to extend DifferenceExport.

### CODE REVIEW

File: modules/data/src/main/java/io/fluo/webindex/data/fluo/UriCountExport.java [hunk 1]
```diff
@@ -15,41 +15,47 @@
...
     }
-    mutations.add(createPageUpdate(uri, seq, newCount));
-
-    return mutations;
+    Map<RowColumn, Bytes> rcMap = new HashMap<>();
+    Bytes linksTo = Bytes.of("" + val.linksTo);
+    rcMap.put(new RowColumn(createTotalRow(uri, val.linksTo), Column.EMPTY), linksTo);
+    String domainRow = getDomainRow(uri);
+    if (domainRow != null) {
+      String cq = String.format("%s:%s", IndexUtil.revEncodeLong(val.linksTo), uri);
```
Comment (keith-turner) [inline modules/data/src/main/java/io/fluo/webindex/data/fluo/UriCountExport.java]: I have found that String.format can be really slow.  I suspect that string concatenation would be much faster in this case.  I also suspect the code would be shorter.

File: modules/data/src/main/java/io/fluo/webindex/data/fluo/DomainExport.java [hunk 2]
```diff
@@ -14,30 +14,29 @@
...
 
   @Override
-  public Collection<Mutation> toMutations(String domain, long seq) {
-    Mutation m = new Mutation("d:" + domain);
+  protected Map<RowColumn, Bytes> generateData(String domain, Long count) {
     if (count == 0) {
-      m.putDelete(Constants.DOMAIN, Constants.PAGECOUNT, seq);
-    } else {
-      m.put(Constants.DOMAIN, Constants.PAGECOUNT, seq, count + "");
+      return Collections.emptyMap();
```
Comment (keith-turner) [inline modules/data/src/main/java/io/fluo/webindex/data/fluo/DomainExport.java]: had to look at this a second to ensure it deletes when new value is zero... I think it does

**Comment by keith-turner** [comment]
+1

Nice to see all that code go away.  I like the name AccumuloExport much better than what it replaced.

File: modules/data/src/main/java/io/fluo/webindex/data/fluo/UriCountExport.java (hunk 1 above)
Comment (mikewalch) [inline modules/data/src/main/java/io/fluo/webindex/data/fluo/UriCountExport.java]: Fixed in 9d42832aa

**Pull request merged**